### ISSUE TITLE
Missing functional Coriolis/beta-plane field constructors: coriolis_fn, beta_param, coriolis_param

### ISSUE THREAD

**Issue by jejjohnson** [jejjohnson/finitevolX#147]
## Background

Somax exposes standalone functional utilities to *construct* Coriolis/beta-plane parameter fields for use in geophysical models:

```python
# somax/_src/operators/functional/coriolis.py

def coriolis_fn(Y, f0=9.375e-5, beta=1.754e-11, y0=None):
    """f = f₀ + β(y - y₀)  — beta-plane approximation"""
    if y0 is None:
        y0 = jnp.mean(Y)
    return f0 + beta * (Y - y0)

def beta_param(lat, omega=OMEGA, radius=R_EARTH):
    """β = (1/R) 2Ω cos θ₀  — from mean latitude"""
    lat = jnp.deg2rad(lat)
    return (2 * omega / radius) * jnp.cos(lat)

def coriolis_param(lat, omega=OMEGA):
    """f₀ = 2Ω sin θ₀  — from mean latitude"""
    lat0 = jnp.mean(jnp.deg2rad(lat))
    return 2.0 * omega * jnp.sin(lat0)
```

Source: [`somax/_src/operators/functional/coriolis.py`](https://github.com/jejjohnson/somax/blob/8dc7229fa57dde0366986b97b6fe7372e3da21e0/somax/_src/operators/functional/coriolis.py)

Also duplicated in [`somax/_src/utils/coriolis.py`](https://github.com/jejjohnson/somax/blob/8dc7229fa57dde0366986b97b6fe7372e3da21e0/somax/_src/utils/coriolis.py).

## Gap

finitevolX already has:
- `Coriolis2D` / `Coriolis3D` — class-based operators that *apply* the Coriolis force to velocity fields
- Physical constants `OMEGA`, `R_EARTH` in [`finitevolx/_src/grid/constants.py`](https://github.com/jejjohnson/finitevolX/blob/917f32d729e483c075d1c8ef35725c8e054e5737/finitevolx/_src/grid/constants.py)

But **no public functional utilities** for *constructing* the Coriolis `f`-field on a grid. The `f` field must be built externally before being passed to `Coriolis2D`, but there is no finitevolX primitive to do this.

## Request

Add public functional utilities to finitevolX for:
- `coriolis_fn(Y, f0, beta, y0=None)` — build a beta-plane `f`-field from y-coordinates
- `beta_param(lat, omega=OMEGA, radius=R_EARTH)` — compute the beta-plane gradient coefficient from latitude
- `coriolis_param(lat, omega=OMEGA)` — compute the reference Coriolis parameter `f₀` from latitude

These should be pure functions (no class state), exported publicly from `finitevolx`.

**Open questions for future clarification:**
- Should `coriolis_fn` accept a full 2D `Y` grid array or just a 1D coordinate vector?
- Should staggered-grid variants be provided (e.g. `f` on T/U/V/X points)?
- Is there a preferred naming convention — `build_coriolis_field`, `beta_plane_coriolis`, etc.?
- Should these live under a new `finitevolx.geo` or `finitevolx.coriolis` submodule, or alongside the existing operator modules?

## References

- Somax: [`coriolis.py`](https://github.com/jejjohnson/somax/blob/8dc7229fa57dde0366986b97b6fe7372e3da21e0/somax/_src/operators/functional/coriolis.py)
- finitevolX constants: [`constants.py`](https://github.com/jejjohnson/finitevolX/blob/917f32d729e483c075d1c8ef35725c8e054e5737/finitevolx/_src/grid/constants.py)
- finitevolX Coriolis operator: [`coriolis.py`](https://github.com/jejjohnson/finitevolX/blob/917f32d729e483c075d1c8ef35725c8e054e5737/finitevolx/_src/operators/coriolis.py)
